LightVox_UnitTests/unittest1.cpp

```diff
@@ -0,0 +1,57 @@
+#include "stdafx.h"
```

Comment:
Might wanna have a better name for the file

Reply:
renamed to `ConcurrentQueueTest.cpp`